runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -414,8 +418,12 @@ public void subscribeLogReplicationListener(@Nonnull LogReplicationListener stre
                                                   @Nonnull String namespace, @Nonnull String streamTag,
                                                   @Nonnull List<String> tablesOfInterest) {
         int uninitializedBufferSize = 0;
-        runtime.getTableRegistry().getLogReplicationUtils().subscribe(streamListener, namespace, streamTag,
-                tablesOfInterest, uninitializedBufferSize, this);
+
+        if (logReplicationUtils == null) {
+            logReplicationUtils = new LogReplicationUtils();
```

Comment:
Why do we instantiate the utils class? 

https://stackoverflow.com/questions/25223553/how-can-i-create-an-utility-class

Reply:
Also, at the end seems we only have one method in the utils, is it really needed?